packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -327,34 +326,6 @@ export enum StorageClass {
   DEEP_ARCHIVE = 'DEEP_ARCHIVE'
 }
 
-/**
- * Used for HTTP expires header, which influences downstream caches. Does NOT influence deletion of the object.
- * @see https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html#SysMetadata
- */
-export class Expires {
```

Comment:
Are we comfortable with this breaking change?

Reply:
thoughts on writing the `@deprecated` tag to the doc string and add `Expiration` to core?